components/Users/UserCard.vue

```diff
@@ -1,29 +1,11 @@
 <template>
-  <v-card class="mx-auto d-flex flex-column">
-    <div class="user-image primary darken-4">
-      <v-responsive :aspect-ratio="1 / 1">
-        <user-image v-if="user.PrimaryImageTag" :id="user.Id" />
-        <div
-          v-if="!user.PrimaryImageTag"
-          class="empty-picture d-flex align-center justify-center"
-        >
-          <v-icon dark size="96">mdi-account</v-icon>
-        </div>
-      </v-responsive>
-    </div>
-    <v-card-title>
-      {{ user.Name }}
-    </v-card-title>
-    <v-card-subtitle class="pb-0 text-capitalize-first-letter">
-      {{ formatDistance(user.LastActivityDate) }}
-    </v-card-subtitle>
-    <v-spacer />
-    <v-card-actions>
-      <v-btn text block color="primary" @click="$emit('connect', user)">
-        {{ $t('login.connect') }}
-      </v-btn>
-    </v-card-actions>
-  </v-card>
+  <div
+    class="user-card mx-auto d-flex flex-column"
+    @click="$emit('connect', user)"
+  >
+    <user-image :size="128" :user="user" class="pointer" />
+    <span class="text-subtitle-1 text-center mt-2">{{ user.Name }}</span>
+  </div>
```

Comment:
Why not use a `<button>`?

Reply:
I can try that this afternoon

Reply:
`<button>` works, but I can get a nice effect with the `plain` attribute of `v-btn`, also I'd need to rework it haha

https://user-images.githubusercontent.com/1619359/107372874-30485080-6ae6-11eb-904c-1c5ad8212fe4.mp4

Reply:
Looks good so far!

Reply:
The video or the current state?